crates/server/src/lib.rs

```diff
@@ -350,7 +351,6 @@ fn create_http_server(
       .wrap(middleware::Compress::default())
       .wrap(cors_config)
       .wrap(TracingLogger::<DefaultRootSpanBuilder>::new())
-      .wrap(ErrorHandlers::new().default_handler(jsonify_plain_text_errors))
```

Comment:
I noticed that cors and compression were not applied to error responses, by reordering middlewares it works properly now.